src/Test.FunctionalTests.BinSkim.Driver/BaselineTestData/Expected/MixedMode_x64_VS2015_Default.exe.sarif

```diff
@@ -55,7 +55,28 @@
             "id": "Warning_NativeWithInsecureStaticLibraryCompilands",
             "arguments": [
               "MixedMode_x64_VS2015_Default.exe",
-              "Microsoft (R) Macro Assembler : masm : 14.0.24123.0 : MSVCRT.lib (_guard_dispatch_.obj)\r\nMicrosoft (R) Optimizing Compiler : c : 19.0.24123.0 : MSVCRT.lib (_cpu_disp_.obj,checkcfg.obj,gs_cookie.obj,gs_support.obj,guard_support.obj,loadcfg.obj,ucrt_detection.obj)\r\nMicrosoft (R) Optimizing Compiler : cxx : 19.0.24123.0 : MSVCMRT.lib (managdeh.obj,ManagedMain.obj,msilexit.obj,mstartup.obj,pureMSILcode.obj)\r\nMicrosoft (R) Optimizing Compiler : cxx : 19.0.24123.0 : MSVCRT.lib (_initsect_.obj,argv_mode.obj,commit_mode.obj,file_mode.obj,initializers.obj,matherr.obj,new_mode.obj,thread_locale.obj,ucrt_stubs.obj,utility.obj,utility_desktop.obj)\r\n"
+              "Microsoft (R) Macro Assembler : masm : 14.0.24123.0 : MSVCRT.lib [MD5] (_guard_dispatch_.obj)\r\nMicrosoft (R) Optimizing Compiler : c : 19.0.24123.0 : MSVCRT.lib [MD5] (_cpu_disp_.obj,checkcfg.obj,gs_cookie.obj,gs_support.obj,guard_support.obj,loadcfg.obj,ucrt_detection.obj)\r\nMicrosoft (R) Optimizing Compiler : cxx : 19.0.24123.0 : MSVCMRT.lib [MD5] (managdeh.obj,ManagedMain.obj,msilexit.obj,mstartup.obj,pureMSILcode.obj)\r\nMicrosoft (R) Optimizing Compiler : cxx : 19.0.24123.0 : MSVCRT.lib [MD5] (_initsect_.obj,argv_mode.obj,commit_mode.obj,file_mode.obj,initializers.obj,matherr.obj,new_mode.obj,thread_locale.obj,ucrt_stubs.obj,utility.obj,utility_desktop.obj)\r\n"
+            ]
+          },
+          "locations": [
+            {
+              "physicalLocation": {
+                "artifactLocation": {
+                  "uri": "file:///Z:/src/Test.FunctionalTests.BinSkim.Driver/BaselineTestData/MixedMode_x64_VS2015_Default.exe",
+                  "index": 0
+                }
+              }
+            }
+          ]
+        },
+        {
+          "ruleId": "BA2004",
+          "ruleIndex": 2,
+          "message": {
+            "id": "Warning_NativeWithInsecureStaticLibraryCompilands",
+            "arguments": [
+              "MixedMode_x64_VS2015_Default.exe",
+              "Microsoft (R) Optimizing Compiler : cxx : 19.0.24215.1 : [directly linked] [No hash value present] (.NETFramework,Version=v4.5.2.AssemblyAttributes.obj)\r\n"
```

Comment:
>AssemblyAttributes [](http://example.com/codeflow?start=144&length=18)

Well! How do you like this? This is exactly the condition reported by a customer. It may be we could/should simply exclude missing hash data for this obj, which looks autogenerated for mixed mode binaries.

Reply:
looks nice.
for a same file we had:
`error` for direct;
`warning`;
question is if for the new one added "no hash"
do we want to use `note`, or same warning`